packages/react-components/react-toast/src/components/Toast.tsx

```diff
@@ -95,9 +91,27 @@ export const Toast: React.FC<ToastProps & { visible: boolean }> = props => {
     }
   }, [play, toastRef]);
 
+  const onEntering = () => {
+    if (!toastRef.current) {
+      return;
+    }
+
+    const element = toastRef.current;
+    element.style.setProperty('--fui-toast-height', `${element.scrollHeight}px`);
```

Comment:
it's impossible to animate to `height: auto`, so using some JS magic and transitions to help here